docs-source/content/samples/simple/domains/soa-domain/_index.md

```diff
@@ -248,32 +160,54 @@ metadata:
 spec:
   # The WebLogic Domain Home
   domainHome: /u01/oracle/user_projects/domains/soainfra
+ 
   # If the domain home is in the image
   domainHomeInImage: false
+ 
   # The WebLogic Server Docker image that the Operator uses to start the domain
   image: "container-registry.oracle.com/middleware/soasuite:12.2.1.3"
+ 
   # imagePullPolicy defaults to "Always" if image version is :latest
   imagePullPolicy: "IfNotPresent"
+ 
   # Identify which Secret contains the credentials for pulling an image
   #imagePullSecrets:
   #- name:
+ 
   # Identify which Secret contains the WebLogic Admin credentials (note that there is an example of
   # how to create that Secret at the end of this file)
   webLogicCredentialsSecret:
     name: soainfra-domain-credentials
+ 
   # Whether to include the server out file into the pod's stdout, default is true
   includeServerOutInPodLog: true
+ 
   # Whether to enable log home
   logHomeEnabled: true
+ 
   # The in-pod location for domain log, server logs, server out, and Node Manager log files
   logHome: /u01/oracle/user_projects/domains/logs/soainfra
+  # An (optional) in-pod location for data storage of default and custom file stores.
+  # If not specified or the value is either not set or empty (e.g. dataHome: "") then the
+  # data storage directories are determined from the WebLogic domain home configuration.
+  dataHome: ""
+ 
+  # Istio service mesh support is experimental.
```

Comment:
Does the Istio support work with a SOA domain?  I'm guessing it does since you've included it here.

Reply:
not sure, removed this until i can test it